packages/web/src/components/nav/desktop/PlaylistLibrary.tsx

```diff
@@ -239,6 +241,110 @@ const PlaylistFolderNavItem = ({
   )
 }
 
+type PlaylistNavItemProps = {
+  playlist: AccountCollection
+  url: string
+  addTrack: (trackId: ID) => void
+  isOwner: boolean
+  onReorder: (
+    draggingId: ID | SmartCollectionVariant,
+    droppingId: ID | SmartCollectionVariant
+  ) => void
+  hasUpdate?: boolean
+  dragging: boolean
+  draggingKind: string
+  onClickPlaylist: (id: ID, hasUpdate: boolean) => void
+  onClickEdit?: (id: ID) => void
+}
+const PlaylistNavItem = ({
+  playlist,
+  hasUpdate = false,
+  url,
+  addTrack,
+  isOwner,
+  onReorder,
+  dragging,
+  draggingKind,
+  onClickPlaylist,
+  onClickEdit
+}: PlaylistNavItemProps) => {
+  const { id, name } = playlist
+  const [isHovering, setIsHovering] = useState(false)
+
+  return (
+    <Droppable
+      key={id}
+      className={navColumnStyles.droppable}
+      hoverClassName={navColumnStyles.droppableHover}
+      onDrop={addTrack}
+      acceptedKinds={['track']}
+      disabled={!isOwner}
+    >
+      <PlaylistNavLink
+        droppableKey={id}
+        playlistId={id}
+        name={name}
+        link={url}
+        to={url}
+        onReorder={onReorder}
+        isActive={() => url === getPathname()}
+        activeClassName='active'
+        className={cn(navColumnStyles.link, {
+          [navColumnStyles.playlistUpdate]: hasUpdate,
+          [navColumnStyles.droppableLink]:
+            isOwner &&
+            dragging &&
+            (draggingKind === 'track' || draggingKind === 'playlist'),
+          [navColumnStyles.disabledLink]:
+            dragging &&
+            ((draggingKind !== 'track' &&
+              draggingKind !== 'playlist' &&
+              draggingKind !== 'library-playlist') ||
+              !isOwner)
+        })}
+        onClick={() => onClickPlaylist(id, hasUpdate)}
+        onMouseEnter={() => {
+          setIsHovering(true)
+        }}
+        onMouseLeave={() => setIsHovering(false)}
+      >
+        <div className={navColumnStyles.playlistLinkContentContainer}>
+          {!hasUpdate ? null : (
+            <div className={navColumnStyles.updateDotContainer}>
+              <Tooltip
+                className={navColumnStyles.updateDotTooltip}
+                shouldWrapContent={true}
+                shouldDismissOnClick={false}
+                mount={null}
+                mouseEnterDelay={0.1}
+                text='Recently Updated'
+              >
+                <div>
```

Comment:
I see this was here before, but why the extra div?

Reply:
it's so the ant tooltip can render properly since it clones the child element and adds some props https://github.com/ant-design/ant-design/issues/15909